rca/people/models.py

```diff
@@ -686,6 +740,33 @@ def name(self):
         parts = (self.student_title, self.first_name, self.last_name)
         return " ".join(p for p in parts if p)
 
+    @property
+    def supervisors(self):
+        if not self.related_supervisor:
```

Comment:
This check is probably redundant, as if there isn't a supervisor it'll return an empty list